cocos2d/cocos/renderer/CCTrianglesCommand.cpp

```diff
@@ -104,4 +104,10 @@ void TrianglesCommand::generateMaterialID()
 //    _materialID = XXH32((const void*)&hashMe, sizeof(hashMe), 0);
 }
 
+void TrianglesCommand::updateMaterialIDWithHash(uint32_t uniformsHash)
```

Comment:
Rename the function so we understand that if will force the batching!
something like enableBatchingWithUniformID

Reply:
renamed to enableBatchingForCustomShader